src/css/css-lib/badge.css

```diff
@@ -0,0 +1,71 @@
+.badge-wrapper {
+  position: relative;
+  display: inline-flex;
+}
+.badge {
+  position: absolute;
+  top: 0;
+  right: 0;
+  border-radius: 0.625rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-width: 1.25rem;
+  width: auto;
+  height: 1.25rem;
+  transform: scale(1) translate(40%, -40%);
+  transform-origin: 100% 0%;
+  font-size: 0.75rem;
+  padding: 0 0.375rem;
+  line-height: 1;
+  font-weight: 600;
+}
+.badge--sm {
+  min-width: 1rem;
+  height: 1rem;
+  font-size: 0.6rem;
+}
+.badge--md {
+  min-width: 1.25rem;
+  height: 1.25rem;
+  font-size: 0.75rem;
+}
+.badge--lg {
+  min-width: 1.5rem;
+  height: 1.5rem;
+  font-size: 0.9rem;
+}
+
+.badge-dot {
```

Comment:
Bade dot on avatar position at bottom right will improve comp. desgin.

Reply:
There's a bottom alignment option already. Will add for left side as well 👍